src/test/java/org/gridsuite/voltageinit/utils/TestUtils.java

```diff
@@ -7,22 +7,22 @@
 
 package org.gridsuite.voltageinit.utils;
 
-import com.google.common.io.ByteStreams;
```

Comment:
revert all the changes

Reply:
why? I hade problems to load resources so I fixed it.